Gemfile

```diff
@@ -8,6 +8,10 @@ rails_version = "#{ENV['RAILS_VERSION'] || 'main'}"
 gem "capybara", "~> 3"
 gem "rails", rails_version == "main" ? { git: "https://github.com/rails/rails", ref: "main" } : rails_version
 
+group :development, :test do
+  gem "appraisal", github: "excid3/appraisal", branch: "fix-bundle-env"
```

Comment:
Is there a reason we shouldn't be using `thoughtbot/appraisal`?

Reply:
Yes because the fix we need is not yet merged: https://github.com/thoughtbot/appraisal/pull/174

Reply:
Is there a way we can keep tabs on where @excid3's fork is at to ensure it's not susceptible to upstream issues with Appraisal?

Reply:
Just FYI, I am skipping appraisal in CI right now and just setting `BUNDLE_GEMFILE` via `matrix`.

https://github.com/excid3/noticed/blob/master/.github/workflows/ci.yml#L28